mlrun/utils/regex.py

```diff
@@ -21,6 +21,11 @@
 
 run_name = label_value
 
+# sparkjob name value format
+# the actual limit is for 63 characters, but due to mlrun and spark operator additions for unique
+# values - the limit is set to 30
```

Comment:
To make life easier for the next one trying to figure out what's going on here, please add description of the exact addition, who adds what when, e.g.
MLRun adds 8 characters to make the run name unique in `KubeResource._get_meta`
Spark operator adds character for the different pods, the maximum is for the X (driver/exec) pod in which the Y suffix is added to the name 
etc...